fq.go

```diff
@@ -0,0 +1,163 @@
+package fq
+
+import (
+	"fmt"
+	"math"
+	"sync"
+	"time"
+)
+
+type FQScheduler struct {
+	lock   *sync.Mutex
+	queues []*Queue
+	vt     uint64
+	C      uint64
+	G      uint64
+	seen   bool
```

Comment:
Why is this field here?

Reply:
This field would not be in the final implementation, it was a 'hack' for testing.  I think that the 'Sleep' for waiting for all Packets makes sense for now until I add a better skeleton around FQ.